### PR TITLE
Added photos to mockUsers and wrote function to create all reviews

### PR DESCRIPTION
**Pull request by ZeMunchkin**

### CODE REVIEW

File: tests/mockDataTests/writeAllReviews.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+});
+
+test('writeAllReviews should write 3-4 reviews for each restaurant', () => {
+  testRestaurants[0].rating = 4;
+  const reviews = writeAllReviews(testRestaurants);
+
+  expect(reviews.length).toBeGreaterThanOrEqual(3);
+});
+
+test('The ratings average to each restaurant\'s overall rating', () => {
```
Comment (MichaelDurfey) [inline tests/mockDataTests/writeAllReviews.test.js]: I'd maybe rename this test to include the average you'd expect it to be just so it's more clear what the test is doing

File: mockData/writeAllReviews.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const generateReview = require('./reviewGenerator.js');
+const helpers = require('./reviewsHelperData.js');
+// const allRestaurants = require('./basicRestaurantData.js');
```
Comment (MichaelDurfey) [inline mockData/writeAllReviews.js]: Not sure if you'll need this later but I'd suggest deleting unused and commented out code

File: tests/mockDataTests/writeAllReviews.test.js (hunk 1 above)
Comment (ZeMunchkin) [reply tests/mockDataTests/writeAllReviews.test.js]: Resolved by additional commit.

File: mockData/writeAllReviews.js (hunk 2 above)
Comment (ZeMunchkin) [reply mockData/writeAllReviews.js]: Will need that line of code when it's time to feed all of these reviews into the database. It's just a reminder to myself.

**Pull request merged**